modules/sharedaddy/sharing-service.php

```diff
@@ -867,6 +867,13 @@ function sharing_display( $text = '', $echo = false ) {
 			$sharing_content .= implode( '', $parts );
 			$sharing_content .= '<li class="share-end"></li></ul>';
 
+			// Link to customization options if user can manage them.
+			if ( current_user_can( 'manage_options' ) ) {
+				$link_text        = __( 'Customize buttons', 'jetpack' );
+				$link_url         = 'https://jetpack.com/redirect/?source=calypso-marketing-sharing-buttons&site=' . Jetpack::build_raw_urls( get_home_url() );
```

Comment:
I would recommend using `\Automattic\Jetpack\Redirect::get_url()` to build that URL.

See PCYsg-pY7-p2 for more info.

I also wonder if we should link to the sharing settings in wp-admin for Jetpack site owners (non-Atomic)?